docs/getting_started/write.md

```diff
@@ -84,6 +84,114 @@ Then, create a file `environment.json` with the following content :
     | `environmentsPath` | `String`                   |           | `.chutney/environments` | where to save the imported environments                  |
     | `force`            | `Boolean`                  |           | `false`                 | if true, locally existing environments files will be overridden.</br>Else they will be kept unchanged |
 
+
+# Declare dataset
+
+You can create a dataset and use it in your scenarios.
+When you declare a dataset, you will provide a name, a description and a tag.
+The name will be the identifier to use in your scenarios
+
+There are 2 types of dataset :
+* Constant dataset which is a set of pairs key/value.
+* Array dataset which is composed by headers and a set of rows.
+
+
+## Constant dataset
+
+The code below show how to use constants in your scenario
+
+``` kotlin
+data class Data(
```

Comment:
I prefer Dataset, it's confisuing otherwise

Reply:
Ok